nodestream/pipeline/extractors/streams/kafka.py

```diff
@@ -17,12 +17,14 @@ def build_consumer_config(
         security_protocol: str,
         group_id: str,
         bootstrap_servers: List[str],
+        **extra_params,
```

Comment:
Why would this not be:
```suggestion
        **kwargs: Any,
```

Reply:
I'm okay with keeping it consistent. I was basing my code changes off of the changes from etwpipeline